src/containers/SpecialTextContainer.js

```diff
@@ -0,0 +1,11 @@
+import {connect} from "react-redux";
+import SpecialText  from "../components/SpecialText";
+
+const mapStateToProps = function (state) {
+  return {
+    text: state.text
```

Comment:
Double check this. Are you sure it should be state.text?
how do you determine what property of state to use?